demo/src/test/java/Y/MilkablesTest.java

```diff
@@ -0,0 +1,81 @@
+package Y;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class MilkablesTest {
+
+    // Not used function, but was added for upgrading animals making old animals still relevant
+    @Test
+    public void upgradeAnimalTest() {
+        Almond almond = new Almond();
+        almond.upgradeAnimal();
+        assertEquals(almond.getMilkValue(), 22);
+    }
+
+    // Might be unessesary testing
```

Comment:
It's slightly unecessary but can be useful to ensure that code when it's refactored still adheres to the original design.